x/peggy/abci.go

```diff
@@ -283,7 +340,8 @@ func (h *BlockHandler) batchSlashing(ctx sdk.Context, params *types.Params) {
 	for _, batch := range unslashedBatches {
 		// SLASH BONDED VALIDTORS who didn't attest batch requests
 		currentBondedSet := h.k.StakingKeeper.GetBondedValidatorsByPower(ctx)
-		confirms := h.k.GetBatchConfirmByNonceAndTokenContract(ctx, batch.BatchNonce, common.HexToAddress(batch.TokenContract))
+		confirms := prepBatchConfirms(ctx, h.k, batch)
+		// confirms := h.k.GetBatchConfirmByNonceAndTokenContract(ctx, batch.BatchNonce, common.HexToAddress(batch.TokenContract))
```

Comment:
Why is this commented out? Do we need it?

Reply:
We don't, removed.